libjob_queue/src/ext_job.c

```diff
@@ -817,7 +809,21 @@ void ext_job_fprintf_config(const ext_job_type * ext_job , const char * fmt , FI
 }
 
 
+static char * ext_job_alloc_std_filename(ext_job_type * ext_job, const char * filetype) {
+  char * new_std_file = util_malloc(strlen(ext_job->name) + strlen(filetype) + 2);
+  strcpy(new_std_file, ext_job->name);
+  strcat(new_std_file, ".");
+  strcat(new_std_file, filetype);
+  return new_std_file;
+}
 
+static void ext_job_set_default_stdout_file(ext_job_type * ext_job) {
+  ext_job->stdout_file = ext_job_alloc_std_filename(ext_job, EXT_JOB_STDOUT);
```

Comment:
This can just be: `ext_job->stdout_file = util_alloc_filename( NULL, ext_job->name, EXT_JOB_STDOUT); `